pasta_eln/inputOutput.py

```diff
@@ -631,3 +647,42 @@ def processNode(graph: list[dict[str,Any]], nodeID:str) -> bool:
         print(f'{prefix}{k:15}: {v}')
   print('\n\nSuccess:', success)
   return
+
+
+def testSignature(fileName:str) -> bool:
```

Comment:
Why define test methods in production code modules?

Reply:
After creating a file, it makes sense to test its validity, or does it not? Also, when importing a eln file from another source, it makes sense to first test it before importing it.